app/src/core/profile/Profile.ts

```diff
@@ -166,4 +169,24 @@ export default class Profile extends EventEmitter {
             }
         });
     }
+
+    catchErrors = new CatchErrors(this._dispatch);
```

Comment:
initilization of catchErrors should be inside a constructor!! And you should create a new class field, named like "_catchErrors"